collectors/gateway/colconfig.yaml

```diff
@@ -54,6 +54,11 @@ exporters:
     tls:
       insecure: true
 
+  otlphttp/jaeger:
+    endpoint: http://jaeger.jaeger-all-in-one-collector.svc.cluster.local:4318
```

Comment:
Is this endpoint correct? Can't seem to find where this namespace is created "jaeger-all-in-one-collector"

Reply:
Fixed